library/include/rocblas_bfloat16.h

```diff
@@ -73,6 +73,11 @@ struct rocblas_bfloat16
         return u.fp32;
     }
 
+    explicit constexpr __host__ __device__ operator double() const
```

Comment:
We do not want to all all kinds of types to convert to -- converting from `rocblas_bfloat16` to `float` should be sufficient, then convert up to `double` or other types.

Reply:
understand I will correct it on third commit "update complex change".

Reply:
`float(x)` should be used to convert `rocblas_bfloat16` to `float`, then it can be converted to other types.